plugin.py

```diff
@@ -46,5 +47,50 @@ def on_pre_server_command(self, params: Mapping[str, Any], done_callback: Callab
             done_callback()
         return True
 
+    def on_server_response_async(self, method: str, response: Response) -> None:
+        if method == 'codeAction/resolve':
+            self.maybe_handle_move_code_action(response)
 
+    def maybe_handle_move_code_action(self, response: Response) -> None:
```

Comment:
Maybe pass `response.result` to this function and type the argument as `CompletionItem`